clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -260,6 +271,33 @@ static cl::opt<bool> AddOpenMPOffloadNotes(
 
 namespace {
 
+auto getValueAsStringRef(const Module *M, Constant *Name) {
```

Comment:
Please add a comment for this helper function. Also, I suppose it can be marked 'static'?

Thanks

Reply:
Comment added.